src/frontend/src/pages/LoginPage/Login.tsx

```diff
@@ -21,7 +19,7 @@ interface LoginProps {
  * Page for unauthenticated users to do login.
  */
 const Login: React.FC<LoginProps> = ({ postLoginRedirect }) => {
-  const [devUserRole, setDevUserRole] = useState<string>(RoleEnum.APP_ADMIN as string);
+  const [devUserId, setDevUserId] = useState(1);
```

Comment:
why is this set to 1 and not 0 as default? don't we have a userId of 1 in the database?

Reply:
because then if you try to login without changing the dropdown at all it won't work